sentry-android/src/main/java/io/sentry/android/AndroidSerializer.java

```diff
@@ -11,7 +13,5 @@ public SentryEnvelope deserializeEnvelope(String envelope) {
   }
 
   @Override
-  public String serialize(SentryEvent event) {
-    return null;
-  }
+  public void serialize(SentryEvent event, Writer writer) throws IOException {}
```

Comment:
why do we need a `Writer` here?

Reply:
Because serializing to a string only to write that string into a writer is non-optimal when you can serialize directly to the writer.

Reply:
Because serializing to a string only to write that string to a writer is kinda suboptimal when you can serialize directly into a writer. E.g. https://github.com/getsentry/sentry-android/pull/14/files#diff-eec907f5d2232bf99fecad94aaf3e4beR100